e2e/tests/cluster/suite.go

```diff
@@ -61,8 +62,9 @@ func SetupClusterLifecycleTest() (*Test, error) {
 	testID := model.NewID()
 	state.TestID = testID
 	logger := logrus.WithFields(map[string]interface{}{
-		"test":   "cluster-lifecycle",
-		"testID": testID,
+		"test":     "cluster-lifecycle",
+		"testID":   testID,
+		"hostname": os.Getenv("HOSTNAME"),
```

Comment:
What is the use case of this `hostname`?

Reply:
I'm not sure whether we can get the Pod Name unless we include the following in the Pod Spec.

```yaml
      env:
        - name: HOSTNAME
          valueFrom:
            fieldRef:
              fieldPath: metadata.name
```

Reply:
BTW, Pod name is already available in logs; `label: pod`.

Reply:
> I'm not sure whether we can get the Pod Name unless we include the following in the Pod Spec.

Just checked and `HOSTNAME` env var is there without any configuration.

> BTW, Pod name is already available in logs; label: pod.

Oh my you're right. I assumed it won't be there since we didn't add it as a field, but prometheus gets it from the k8s cluster anyway. Have been reading logs all week and didn't notice 😓  Reverting this then!